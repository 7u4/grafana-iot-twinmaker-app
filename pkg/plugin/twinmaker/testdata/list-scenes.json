{
    "NextToken": null,
    "SceneSummaries": [
        {
            "Arn": "arn:aws:iottwinmaker:us-east-1:166800769179:workspace/CookieFactory-11-16/scene/CookieFactory",
            "ContentLocation": "s3://roci-workspace-cookiefactory-11-16-166800769179/CookieFactory.json",
            "CreationDateTime": "2021-11-16T18:53:37.301Z",
            "Description": null,
            "SceneId": "CookieFactory",
            "UpdateDateTime": "2021-11-16T18:53:37.301Z"
        }
    ]
}