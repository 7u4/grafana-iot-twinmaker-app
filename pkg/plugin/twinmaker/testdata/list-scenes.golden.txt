ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] {
    "custom": {}
}
Name: 
Dimensions: 4 Fields by 1 Rows
+-----------------------------------------------------------------------------------------------+-----------------------------------+-------------------+-----------------+
| Name: arn                                                                                     | Name: created                     | Name: description | Name: sceneId   |
| Labels:                                                                                       | Labels:                           | Labels:           | Labels:         |
| Type: []*string                                                                               | Type: []time.Time                 | Type: []*string   | Type: []*string |
+-----------------------------------------------------------------------------------------------+-----------------------------------+-------------------+-----------------+
| arn:aws:iottwinmaker:us-east-1:166800769179:workspace/CookieFactory-11-16/scene/CookieFactory | 2021-11-16 18:53:37.301 +0000 UTC | null              | CookieFactory   |
+-----------------------------------------------------------------------------------------------+-----------------------------------+-------------------+-----------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////aAIAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEDAAoADAAAAAgABAAKAAAACAAAAIAAAAADAAAATAAAACgAAAAEAAAAHP7//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAAA8/v//CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAFz+//8IAAAAGAAAAA0AAAB7ImN1c3RvbSI6e319AAAABAAAAG1ldGEAAAAABAAAAFQBAADYAAAAYAAAAAQAAADO/v//FAAAADwAAAA8AAAAAAAFATgAAAABAAAABAAAALz+//8IAAAAEAAAAAcAAABzY2VuZUlkAAQAAABuYW1lAAAAAAAAAAC4/v//BwAAAHNjZW5lSWQAJv///xQAAABAAAAAQAAAAAAABQE8AAAAAQAAAAQAAAAU////CAAAABQAAAALAAAAZGVzY3JpcHRpb24ABAAAAG5hbWUAAAAAAAAAABT///8LAAAAZGVzY3JpcHRpb24AAAASABgAFAAAABMADAAAAAgABAASAAAAFAAAADwAAABEAAAAAAAACkQAAAABAAAABAAAAIj///8IAAAAEAAAAAcAAABjcmVhdGVkAAQAAABuYW1lAAAAAAAAAAAAAAYACAAGAAYAAAAAAAMABwAAAGNyZWF0ZWQAAAASABgAFAATABIADAAAAAgABAASAAAAFAAAAEAAAABEAAAAAAAFAUAAAAABAAAADAAAAAgADAAIAAQACAAAAAgAAAAMAAAAAwAAAGFybgAEAAAAbmFtZQAAAAAAAAAABAAEAAQAAAADAAAAYXJuAAAAAAD/////SAEAABQAAAAAAAAADAAWABQAEwAMAAQADAAAAJgAAAAAAAAAFAAAAAAAAAMDAAoAGAAMAAgABAAKAAAAFAAAAMgAAAABAAAAAAAAAAAAAAALAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACAAAAAAAAAAIAAAAAAAAAGAAAAAAAAAAaAAAAAAAAAAAAAAAAAAAAGgAAAAAAAAACAAAAAAAAABwAAAAAAAAAAgAAAAAAAAAeAAAAAAAAAAIAAAAAAAAAIAAAAAAAAAAAAAAAAAAAACAAAAAAAAAAAAAAAAAAAAAgAAAAAAAAAAIAAAAAAAAAIgAAAAAAAAAEAAAAAAAAAAAAAAABAAAAAEAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAABAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAXQAAAGFybjphd3M6aW90dHdpbm1ha2VyOnVzLWVhc3QtMToxNjY4MDA3NjkxNzk6d29ya3NwYWNlL0Nvb2tpZUZhY3RvcnktMTEtMTYvc2NlbmUvQ29va2llRmFjdG9yeQAAAECP/QybG7gWAAAAAAAAAAAAAAAAAAAAAAAAAAANAAAAQ29va2llRmFjdG9yeQAAABAAAAAMABQAEgAMAAgABAAMAAAAEAAAACwAAAA4AAAAAAADAAEAAAB4AgAAAAAAAFABAAAAAAAAmAAAAAAAAAAAAAAAAAAAAAAACgAMAAAACAAEAAoAAAAIAAAAgAAAAAMAAABMAAAAKAAAAAQAAAAc/v//CAAAAAwAAAAAAAAAAAAAAAUAAAByZWZJZAAAADz+//8IAAAADAAAAAAAAAAAAAAABAAAAG5hbWUAAAAAXP7//wgAAAAYAAAADQAAAHsiY3VzdG9tIjp7fX0AAAAEAAAAbWV0YQAAAAAEAAAAVAEAANgAAABgAAAABAAAAM7+//8UAAAAPAAAADwAAAAAAAUBOAAAAAEAAAAEAAAAvP7//wgAAAAQAAAABwAAAHNjZW5lSWQABAAAAG5hbWUAAAAAAAAAALj+//8HAAAAc2NlbmVJZAAm////FAAAAEAAAABAAAAAAAAFATwAAAABAAAABAAAABT///8IAAAAFAAAAAsAAABkZXNjcmlwdGlvbgAEAAAAbmFtZQAAAAAAAAAAFP///wsAAABkZXNjcmlwdGlvbgAAABIAGAAUAAAAEwAMAAAACAAEABIAAAAUAAAAPAAAAEQAAAAAAAAKRAAAAAEAAAAEAAAAiP///wgAAAAQAAAABwAAAGNyZWF0ZWQABAAAAG5hbWUAAAAAAAAAAAAABgAIAAYABgAAAAAAAwAHAAAAY3JlYXRlZAAAABIAGAAUABMAEgAMAAAACAAEABIAAAAUAAAAQAAAAEQAAAAAAAUBQAAAAAEAAAAMAAAACAAMAAgABAAIAAAACAAAAAwAAAADAAAAYXJuAAQAAABuYW1lAAAAAAAAAAAEAAQABAAAAAMAAABhcm4AkAIAAEFSUk9XMQ==
